android/app/build.gradle

```diff
@@ -83,7 +94,7 @@ project.ext.react = [
 
 apply from: '../../node_modules/react-native-unimodules/gradle.groovy'
 apply from: "../../node_modules/react-native/react.gradle"
-apply from: "../../node_modules/expo-updates/expo-updates.gradle"
```

Comment:
is this supposed to be removed?

Reply:
YOURE SUPPOSED TO BE REMOVED.

Reply:
Also, yes. This was intentional. I changed the path because of an update in the package files 😄